Sources/Filters/Sources/Arrow2DSource/index.js

```diff
@@ -70,7 +70,24 @@ function vtk6PointsArrow(publicAPI, model) {
   points[16] = model.height * 0.1 - thickOp - offsetOp - centerOffsetOp;
   points[17] = 0.0;
 
-  const cells = Uint8Array.from([6, 0, 1, 2, 3, 4, 5]);
+  const cells = Uint8Array.from([
```

Comment:
Here's a neat trick to make this more "prettier": put `// prettier-ignore` on the previous line, and then you can format this array like the following:

```
// prettier-ignore
const cells = Uint8Array.from([
  3, 0, 1, 5,
  3, 1, 4, 5,
 ...
```

Reply:
Sweet, way cleaner thank you!